mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/kb-credentials.feature

```diff
@@ -5,7 +5,157 @@ Feature: KB Credentials
     * callonce login testUser
     * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
 
-  Scenario: Get all KB Credentials
-    Given path '/eholdings/kb-credentials'
-    When method GET
-    Then status 200
\ No newline at end of file
+ ################ Positive test cases ################
+  @Undefined
+  Scenario: GET /eholdings/kb-credentials should return status 200 and json file
```

Comment:
1. No JSON file is returning by any endpoint of the API.
2. Please change the names of scenarios to be more specific. Like "Should return all existed KB credentials" or "Should create new valid KB credentials"